torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -2778,30 +2778,18 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
     case aten::sum:
     case aten::softmax:
     case aten::log_softmax:
-    case aten::conv2d: {
-      std::vector<ArgValue> argInputs;
-      for (auto inp : inputs) {
-        argInputs.push_back(toArg(inp));
-      }
-      auto outputType = findDtypeForValue(v->node()->output());
-      std::vector<ExprHandle> outputShape = {};
-      // shape inference not implemented for sum
```

Comment:
> // shape inference not implemented for sum

Do you know why this was done before? Is it okay to remove this check now, as your change is doing?

Reply:
We're not using shape inference anyway (ie. `aten::sum` is not any special here), the test was just missing `requires_grad=0` in its listing. Previously we did use TE shape inference and that's probably this check was added.